samples/snippets/src/test/java/com/example/bigquery/UpdateTableDMLIT.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+import static com.google.common.truth.Truth.assertThat;
+import static junit.framework.TestCase.assertNotNull;
+
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.LegacySQLTypeName;
+import com.google.cloud.bigquery.Schema;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class UpdateTableDMLIT {
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  private static final String BIGQUERY_DATASET_NAME = System.getenv("BIGQUERY_DATASET_NAME");
```

Comment:
You are using `getenv` here, but also have requireEnvVar below. Maybe use `requireEnvVar` here but have it to return a value?

Reply:
this is how we have been handling this in all the samples in BigQuery... is there a new standard?

Reply:
You're right, this seems to be the same as being used in other places. However, I think maybe this pattern might be improved if we did something like this instead;

```java
  private static final String BIGQUERY_DATASET_NAME = requireEnvVar("BIGQUERY_DATASET_NAME");

  private static string requireEnvVar(String varName) {
   String value =  System.getenv(varName);
    assertNotNull(
        "Environment variable " + varName + " is required to perform these tests.",
        value);
     return value;
  }
```

This way we only have to specify "BIGQUERY_DATASET_NAME" in one pace. 

(consider this one a nit)

Reply:
okay makes sense - done